controllers/networkinterface_controller.go

```diff
@@ -51,7 +51,7 @@ func getIP(ipFamily corev1.IPFamily, ipFamilies []corev1.IPFamily, ips []metalne
 			return ips[i].Addr
 		}
 	}
-	// If an IP is not specified defined it as "::" or "0.0.0.0" to disable that ip family on dp-service side
+	// If an IP is not specified then sepcify it, as "::" or "0.0.0.0" to disable that ip family on dp-service side
```

Comment:
I assume it shall be "If an IP is not specified, then specify it as "::" or "0.0.0.0" to disable that ip family on dp-service side".